### ISSUE TITLE
Updates stops after some time

### ISSUE THREAD

**Issue by GoogleCodeExporter** [max99x/pagemon-chrome-ext#74]
```
What steps will reproduce the problem?
I use the extension to monitor about 10 channels on youtube, at intervals of 
2-5 minutes. The detection works well for some time but after an interval 
varying from minutes to several hours, the extension stops the upgrading. After 
this point even though I try to update manually, it does not. The only way back 
to work is to disable and re-enable the extension, or restart the program. When 
everything is back to normal for some time.

I have had this problem for some time and realize that the main factor that 
causes the failure is the heavy use of my connection. So imagine that the 
failure may have something to do with attempts to update suffer with timeouts. 
So to reproduce the problem is important to test in conditions of connection 
unavailable for some time.

What is the expected output? What do you see instead?
Some form for the extension retry the update when realize that the desired 
interval has passed.

What version of the Chrome are you using? On what operating system?
Google Chrome 7.0.517 in Win7-64. But the problem ever happens since i start to 
use the extension in version 5.

Please provide any additional information below.
Im using custom mode with selector in monitor, i tried without, but the problem 
remained.
```

Original issue reported on code.google.com by `gandalf....@gmail.com` on 11 Sep 2010 at 10:18

**Comment by GoogleCodeExporter** [comment]
```
I will need you to post the contents of the console after the problem occurs. 
To get to the console, open the Chrome Extensions page and enable developer 
mode, then under Page Monitor -> Inspect active views click on background.htm.
```

Original comment by `max99x` on 11 Sep 2010 at 12:14
* Added labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Thanks for replying. This is the error on console:
______
Uncaught Error: INVALID_STATE_ERR: DOM Exception 11
check.$.ajax.completescripts/background.js:5
dlib/jquery-1.4.1.js:121
c.extend.ajax.g.w.onreadystatechangelib/jquery-1.4.1.js:127
c.extend.ajaxlib/jquery-1.4.1.js:127
______
At least twice before the crash a warning appears "Watchdog recovered a lost 
timeout."
```

Original comment by `gandalf....@gmail.com` on 12 Sep 2010 at 9:13

**Comment by GoogleCodeExporter** [comment]
Original comment by `max99x` on 12 Sep 2010 at 5:35
* Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
Original comment by `max99x` on 21 Sep 2010 at 7:02
* Changed state: **Fixed**
